web/themes/contrib/civictheme/build.js

```diff
@@ -375,15 +375,23 @@ function cli() {
 }
 
 function lintExclusions() {
+  const lintExclusionPaths = [
+    fullPath('./storybook-static/**/*.js'),
+    fullPath('./dist/**/*.js'),
```

Comment:
We also missed some paths of JS we want to exclude

Reply:
Just an idea - can this build script come from a package so it would be managed as upstream dependency?